test_scripts/API/AutoCompleteList/017_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_Invalid_response_on_Mobile.lua

```diff
@@ -0,0 +1,77 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = false, resultCode = "GENERIC_ERROR"> to mobile app
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param param to HMI
```

Comment:
@Aleksandriys,
1. Description currently looks like "expected result". Please try to use description describing case: "Case when ..."
2. please update description with particular resultCode from HMI (please specify "GENERIC_ERROR") and not <resultCode_received_from _HMI>